lib/services/db-operations.js

```diff
@@ -80,6 +80,23 @@ function splitCascadingData(uri, data) {
   return ops;
 }
 
+/**
+ * Adds user info onto each op object
+ *
+ * @param {Object} locals
+ * @returns {Function}
+ */
+function addUserInfoToOp(user) {
+  function addUserProp(op) {
+    op.user = user;
```

Comment:
Just a note that we'll need to stringify this as well if we do it in the [addOp function](https://github.com/clay/amphora/pull/697/files#diff-1743826468bdf1b419b84cdd1148fa5a882add8b21ab7cb731371f66a885d2dfR39)

Reply:
we should be ok without stringify! thanks for bearing with me